src/ripple/overlay/impl/Manifest.cpp

```diff
@@ -206,6 +205,25 @@ ManifestCache::configManifest (
     }
 }
 
+boost::optional<std::string>
+ManifestCache::member (PublicKey const& identity) const
+{
+    std::lock_guard<std::mutex> lock (mutex_);
+
+    auto ret = map_.find (identity);
+
+    if (ret != map_.end())
+        return ret->second.comment;
+
+    return boost::none;
+}
+
+bool
+ManifestCache::trusted (PublicKey const& identity) const
+{
+    return static_cast<bool> (member(identity));
```

Comment:
No, please don't change `ValidatorList`. Leave it as it - there's a reason why `ValidatorList::member` returns a `boost::optional<std::string>`: it saves a double lookup in a hot codepath.